specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -802,16 +802,28 @@
           "type": "string",
           "description": "Version of the guest configuration."
         },
-        "contentUri": {
-          "readOnly": true,
+        "contentUri": {          
           "type": "string",
           "description": "Uri of the storage where guest configuration package is uploaded."
         },
-        "contentHash": {
-          "readOnly": true,
+        "contentHash": {          
           "type": "string",
           "description": "Combined hash of the guest configuration package and configuration parameters."
         },
+		"assignmentType": {
```

Comment:
If this `assignmentType` property is same with the [`configurationMode`](https://github.com/Azure/azure-rest-api-specs/blob/31e74a689702c6faf16187cc16921566cb20aff6/specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json#L857) property, does it mean the `configurationMode` property needs to be removed in this PR?

Reply:
We never used configurationMode.
But since we already have it in swagger spec - we cannot delete it. BreakingChange team wont allow us to remove it.
I will work with my team and come up with a plan to remove the configurationMode section. Right now it wont harm it.